src/markup/codemirror/create.ts

```diff
@@ -112,7 +114,7 @@ export function createCodemirror(params: CreateCodemirrorParams) {
             ...defaultKeymap,
             ...historyKeymap,
         ]),
-        autocompletion(),
+        autocompletion(autocompletionConfig),
         yfmLang(),
```

Comment:
We [added an option](https://github.com/gravity-ui/markdown-editor/pull/315/files#diff-3918eac2ddd197df4e93220958532f747d2ef57c4da6432a565c0737af7db140R138) to the yfmHtml plugin. 

Now you can add your function to autocomplete things. For example, look at how `mdAutocomplete` [works](https://github.com/gravity-ui/markdown-editor/pull/315/files#diff-3918eac2ddd197df4e93220958532f747d2ef57c4da6432a565c0737af7db140R88).

For example you can write a function for suggest with `CompletionsSource` interface from `@codemirror/autocomplete` and add it to options in `yfmLang`

Reply:
great!